scania-digital-design-system/sdds
# Dropdown label inside bug

**Describe pull-request**  
Dropdown with label inside is not rendering the placeholder text.

**Solving issue**  
Github issue #440 and Azure ticket [AB#2321](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2321)

**How to test**  
_Add description how to test if possible_
1. Go to storybook
2. Check dropdown label inside and see if it looks correct
